sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/models/ConsistentReadControl.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.file.datalake.models;
+
+/**
+ * Defines values to indicate what strategy the SDK should use when reading from a blob to ensure the view of the data
+ * is consistent and not changed during the read.
+ * {@link #NONE}
+ * {@link #ETAG}
+ */
+public enum ConsistentReadControl {
```

Comment:
Should this consistency control graduate into `azure-storage-common` as it should be re-usable in Blobs and maybe Files if we choose to add open read/write functionality there as well

Reply:
The issue is that the blob package already has their own copy of this. So if we put one in common then there will be confusion in the blobs package. Something we'd need to have caught before that API GA'd.

Reply:
Datalake takes dependency on blobs directly? Don't we end up having both types anyway on class path ?

Reply:
I may be wrong but I think we go out of our way to avoid using blob types in the datalake public API.